keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -509,6 +508,9 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+If we have a `V+1` version of api-server(`V` refers to the kubernetes version we support this feature), and a `V` version scheduler,
+we'll just ignore `NodeInclusionPolicy` with no side effects.
```

Comment:
This is actually N/A, as kubelet doesn't change.

Scheduler is usually expected to have the same version as apiserver.